Source/SwiftLintFramework/Rules/Idiomatic/ExplicitInitRule.swift

```diff
@@ -40,14 +40,64 @@ public struct ExplicitInitRule: SubstitutionCorrectableASTRule, ConfigurationPro
               obs2,
               resultSelector: { MyType.init($0, $1) }
             ).asMaybe()
+            """),
+            Example("""
+            let int = Int
+            .init(1.0)
+            """),
```

Comment:
Can you please exclude these examples from the generated documentation?

```suggestion
            """).excludeFromDocumentation(),
```